### PR TITLE
exported function StartControllers should have comment

### PR DESCRIPTION
**Pull request by StudyNick**
**What this PR does / why we need it**:
exported function StartControllers should have comment

 "k8s.io/kubernetes/pkg/cloudprovider/providers"  alias _ is not used, suggest to delete

### CODE REVIEW

**Comment by StudyNick** [comment]
I signed it!

File: cmd/kube-controller-manager/app/plugins.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 
 	// Cloud providers
 	"k8s.io/kubernetes/pkg/apis/componentconfig"
-	_ "k8s.io/kubernetes/pkg/cloudprovider/providers"
+
```
Comment (lavalamp) [inline cmd/kube-controller-manager/app/plugins.go]: Pretty sure this is necessary (we need the init() method to get called).

File: cmd/kube-controller-manager/app/controllermanager.go [hunk 2]
```diff
@@ -220,6 +220,9 @@ func Run(s *options.CMServer) error {
 	panic("unreachable")
 }
 
+// StartControllers is start controller (replication controller,node controller,resourceQuota controller,
+// namespace controller,serviceAccount controller,token controller,server controller,
+// endpoint controller) in Controller manager
```
Comment (lavalamp) [inline cmd/kube-controller-manager/app/controllermanager.go]: I suggest "StartControllers starts all the controllers."

I do not recommend putting the list in the comment, since it will likely become quickly out of date.

Additionally--sorry to be nitpicky--but if you do want to have the list, there need to be spaces after the commas :)

File: pkg/credentialprovider/aws/aws_credentials.go [hunk 3]
```diff
@@ -34,6 +34,7 @@ import (
 // and credentialprovider.
 var AWSRegions = [...]string{
 	"us-east-1",
+	"us-east-2",
```
Comment (lavalamp) [inline pkg/credentialprovider/aws/aws_credentials.go]: @mikedanese was this deliberately omitted before?

**Review by lavalamp — CHANGES_REQUESTED** [review]
sorry for the delay, I was OOO most of last month.

**Comment by lavalamp** [comment]
ok to test

File: pkg/credentialprovider/aws/aws_credentials.go (hunk 3 above)
Comment (mikedanese) [inline pkg/credentialprovider/aws/aws_credentials.go]: Might not have existed when this list was aggregated. @justinsb PTAL. This is unrelated to the PR and should probably be left out.

Comment (justinsb) [inline pkg/credentialprovider/aws/aws_credentials.go]: It's been merged already ... surprised this isn't a conflict.

https://github.com/kubernetes/kubernetes/blob/master/pkg/credentialprovider/aws/aws_credentials.go

@StudyNick it's probably a good idea to do one PR per "idea" - even if in this case it means 3 PRs.  Not least because then I could have merged your us-east-2 PR and it would have gone into k8s sooner ;-)
